app/scheduling/controllers/SchedulingCtrl.js

```diff
@@ -217,13 +217,14 @@ schedulingApp.controller('SchedulingCtrl', ['$scope', '$rootScope', '$routeParam
 			schedulingActionCreators.createSection(section);
 		};
 
+		// Return true if the user does not have write access
 		$scope.isLocked = function () {
+			// Keep UI locked while state is still loading
 			if (!$scope.view.state) { return true; }
-			var term = $scope.view.state.uiState.term;
+
 			var hasAuthorizedRole = $scope.sharedState.currentUser.isAdmin() ||
 				$scope.sharedState.currentUser.hasRole('academicPlanner', $scope.sharedState.workgroup.id);
-			// Return true if the term is locked or the user has not write access
-			return term ? term.isLocked() || !(hasAuthorizedRole) : true;
+			return !(hasAuthorizedRole);
```

Comment:
What do the parentheses do?

Reply:
Its a technique I've seen elsewhere to improve clarity, with !hasAuthorizedRole its easier to miss the exclamation, whereas if I write it !(hasAuthorizedRole) it stands out better. Which do you prefer?

Reply:
Oh, I see. I was curious if it was some Javascript "coerce into another type" sort of thing.

Looks good, leave as is.